internal/policy/operator/deployable_by_olm.go

```diff
@@ -106,6 +107,14 @@ func WithCSVTimeout(csvTimeout time.Duration) Option {
 	}
 }
 
+// WithSubscriptionTimeout overrides the default subscriptionTimeout, for subscriptions that
+// additional time to install.
```

Comment:
NIT:

```suggestion
// WithSubscriptionTimeout customizes how long to wait for a subscription to become healthy.
```

Or similar.

Reply:
This is more readable, I'll edit the CSV language in another PR as well.